public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
We could also make the links black but keep their hover state, or even have the whole bubble change to a different colour on hover. Light gray background works too, I think I've just become so accustomed to `#d0d0d0` :)